File: c/PRAK110-2510817210027-Ahmad_Tibrizi.c
#include <stdio.h>
#include <math.h>

void main() {
    int alas = 5;
    int tinggi = 12;
    double sisi_miring = sqrt(alas * alas + tinggi * tinggi);
    int keliling = alas + tinggi + (int)sisi_miring;
    int luas = (alas * tinggi) / 2;
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", alas);
    printf("Sisi B = %d cm\n", tinggi);
    printf("Sisi C = %.0f cm\n", sisi_miring);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", luas);
}

File: python/PRAK110-2510817210027-Ahmad_Tibrizi.py
import math
alas = 5
tinggi = 12
sisi_miring = math.sqrt(alas**2 + tinggi**2)
keliling = alas + tinggi + sisi_miring
luas = (alas * tinggi) / 2
print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm\n")
print("Jawab :")
print(f"Sisi A = {alas} cm")
print(f"Sisi B = {tinggi} cm")
print(f"Sisi C = {int(sisi_miring)} cm")
print(f"Keliling = {int(keliling)} cm")
print(f"Luas = {int(luas)} cm")